framework/common/src/main/java/edu/kit/kastel/mcse/ardoco/core/api/textextraction/NounMapping.java

```diff
@@ -64,7 +66,7 @@ public interface NounMapping {
      */
     ImmutableList<Integer> getMappingSentenceNo();
 
-    ImmutableList<Phrase> getPhrases();
+    ImmutableSortedSet<Phrase> getPhrases();
```

Comment:
I dont think so. In my understanding conceptually a phrase should only exist once, since it is a specific sequence of words from the text. I think I changed this for consistency reasons, since words are also stored in ImmutableSortedSets. I also faintly remember this being the reason for some rather large slowdown when using the ArDoCo Util Comparators functions which I discovered when I did some profiling to mitigate performance issues. Do you have anything in particular in mind that I should check regarding whether or not it broke?